src/components/Header/index.jsx

```diff
@@ -2,15 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import { Button, Icon, Dropdown } from '@edx/paragon';
+import { configuration } from '../../config';
 
 import SidebarToggle from '../../containers/SidebarToggle';
 import Img from '../Img';
 
 import apiClient from '../../data/apiClient';
 
-import EdxLogo from '../../images/edx-logo.png';
 import './Header.scss';
 
+const logo = configuration.LOGO_URL;
```

Comment:
nit: It feels a bit odd to create this variable outside of the component. Any reason not to pull this line into the `renderLogo()` function?